torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -442,12 +442,19 @@ void Reducer::mark_variable_ready_sparse(size_t variable_index) {
   });
 }
 
-std::vector<at::Tensor> Reducer::get_bucket_tensors() const {
+std::vector<c10d::GradBucket> Reducer::get_bucket_tensors() const {
```

Comment:
Can we rename this function as well? Since we no longer directly return tensors, `get_grad_buckets` should be a better name.